src/main/java/umc/spring/web/controller/StoreController.java

```diff
@@ -0,0 +1,68 @@
+package umc.spring.web.controller;
+
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.Parameters;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import umc.spring.Vaildator.annotation.CheckPage;
+import umc.spring.apiPayload.ApiResponse;
+import umc.spring.repository.ReviewRepository.ReviewRepository;
+import umc.spring.service.ReviewService.ReviewQueryServiceImpl;
+import umc.spring.service.StoreService.StoreQueryServiceImpl;
+import umc.spring.web.controller.dto.ReviewDTO.ReviewRequestDTO;
+import umc.spring.web.controller.dto.ReviewDTO.ReviewResponseDTO;
+import umc.spring.web.controller.dto.StoreDTO.StoreRequestDTO;
+import umc.spring.web.controller.dto.StoreDTO.StoreResponseDTO;
+
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping("/stores")
+public class StoreController {
+
+    private final StoreQueryServiceImpl storeQueryService;
+
+
+    @PostMapping("{regionId}")
+    public ApiResponse<StoreResponseDTO.JoinResultDTO>
```

Comment:
아래 특정 가게 리뷰 목록 조회 API처럼 Swagger를 통해  문서화를 추가해주면 더 좋을 것 같아요!!
```suggestion
    @PostMapping("{regionId}")
    @Operation(summary = "가게 등록 API", description = "지역 ID와 가게 정보를 받아 새로운 가게를 등록합니다.")
    @ApiResponses({
        @ApiResponse(responseCode = "200", description = "가게 등록 성공"),
    })
    public ApiResponse<StoreResponseDTO.JoinResultDTO>
```